ms-engine/src/main/java/net/trajano/ms/engine/SpringJaxRsHandler.java

```diff
@@ -239,11 +242,14 @@ public SpringJaxRsHandler(final ConfigurableApplicationContext baseApplicationCo
                 LOG.debug("{} => @Path {}", baseUri + clazz.getAnnotation(Path.class).value(), clazz);
                 applicationContext.register(clazz);
             });
-            reflections.getTypesAnnotatedWith(Provider.class).forEach(clazz -> applicationContext.register(clazz));
+            reflections.getTypesAnnotatedWith(Provider.class).forEach(applicationContext::register);
+            reflections.getTypesAnnotatedWith(RequestScoped.class).forEach(applicationContext::register);
+            reflections.getTypesAnnotatedWith(ApplicationScoped.class).forEach(applicationContext::register);
         } else {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)